src/components/Sum.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Sum = ({ a, b }) => (
+  <p>
+    Sum of
+    {' '}
+    {a}
+    {' '}
+    and
+    {' '}
+    {b}
+    {' '}
+    is
+    {' '}
+    {a + b}
+  </p>
+);
+
+Sum.defaultProps = {
+  a: 2,
+  b: 1,
+};
```

Comment:
Don't set default value as some random values. Either set them as 0 or make'em required in `propTypes`.